sonar-orchestrator/src/main/java/com/sonar/orchestrator/build/ScannerForMSBuildInstaller.java

```diff
@@ -58,54 +60,61 @@ public ScannerForMSBuildInstaller(Locators locators) {
   }
 
   /**
-   * Installs an ephemeral Scanner for MSBuild and returns the path to the exe to execute
+   * Installs an ephemeral Scanner for .NET and returns the path to the exe to execute.
    */
   public File install(@Nullable Version scannerVersion, @Nullable Location location, File toDir, boolean useDotNetCore) {
-    if (location != null) {
-      return installFromLocation(location, toDir, useDotNetCore);
-    } else if (scannerVersion != null) {
-      return install(scannerVersion, toDir, useDotNetCore);
+    if (location == null) {
+      scannerVersion = scannerVersion == null
```

Comment:
Yeah, indentation would need to be only one level to the right (2 spaces). But I think it would be either to read if:
- either write it on one line, it's simple enough anyway:
```java
scannerVersion = (scannerVersion == null) ? Version.create(DEFAULT_SCANNER_VERSION) : scannerVersion;
```
- or even make the intent clearer with a function name, something like:
```java
scannerVersion = getDefaultScannerVersionIfNull(scannerVersion);
  ...
private String getDefaultScannerVersionIfNull(@Nullable String scannerVersion) {
  return (scannerVersion == null) ? Version.create(DEFAULT_SCANNER_VERSION) : scannerVersion;
  }
```